src/Checker/ReviewableOrder/OrderFulfilledReviewableOrderChecker.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusReviewPlugin\Checker\ReviewableOrder;
+
+use Sylius\Component\Core\Model\OrderInterface;
+
+final class OrderFulfilledReviewableOrderChecker implements ReviewableOrderCheckerInterface
+{
+    public function check(OrderInterface $order): ReviewableOrderCheck
+    {
+        if ($order->getState() === OrderInterface::STATE_FULFILLED) {
```

Comment:
Let's add an array of 'reviewable states' in the contructor and make those configurable in the plugin configuration